docs/src/adoc/index.adoc

```diff
@@ -4040,12 +4032,14 @@ TODO:
 * HandlerDecorators - Apply Handler decorations base on any criteria, not just
   annotations.
 ** SqlMethodAnnotatedHandlerDecorator
+////
 
 === Extensions
 
 Jdbi supports generic extensions. SQL Objects are just one implementation of
 that spec. If SQL Objects aren't doing it for you, you can create your own!
 
+////
```

Comment:
I would comment out the entire section--the intro above is sort of pointless by itself.